### PR TITLE
Deploy

### PR DESCRIPTION
**Pull request by vitalypuzanov**

### CODE REVIEW

File: components/Card/Card.js [hunk 1]
```diff
@@ -1,29 +1,30 @@
 import React from 'react';
 import classes from './Card.module.css';
+import Image from 'next/dist/client/image';
+
+import {useRouter} from 'next/router';
```
Comment (AntonovaNastya) [inline components/Card/Card.js:3]: move to 2 line

File: components/Card/Card.js [hunk 2]
```diff
@@ -1,29 +1,30 @@
 import React from 'react';
 import classes from './Card.module.css';
```
Comment (AntonovaNastya) [inline components/Card/Card.js]: should be the latest import

File: components/Card/Card.js [hunk 3]
```diff
@@ -1,29 +1,30 @@
...
+import Image from 'next/dist/client/image';
+
+import {useRouter} from 'next/router';
+function Card(props) {
+  const router = useRouter();
+  const {image, title, description, price} = props.cards;
+  console.log;
```
Comment (AntonovaNastya) [inline components/Card/Card.js]: !

File: components/Card/Card.js [hunk 4]
```diff
@@ -1,29 +1,30 @@
...
+
+  function showDetailsHandler() {
+    router.push('/' + props.id);
```
Comment (AntonovaNastya) [inline components/Card/Card.js]: router.push(props.id);

File: components/Card/Card.module.css [hunk 5]
```diff
@@ -1,22 +1,32 @@
...
   align-items: center;
-  border: black;
-  border-width: 0.5px;
-  border-top: 1 px solid;
-  border-left: 1 px solid;
-  border-right: 1 px solid;
-  border-color: transparent;
   cursor: pointer;
+
+  /* max-width: 100%;
```
Comment (AntonovaNastya) [inline components/Card/Card.module.css]: !!!!!!!!!!!

File: components/Auth/Authform.js [hunk 6]
```diff
@@ -0,0 +1,96 @@
+// import {useState, useRef} from 'react';
```
Comment (AntonovaNastya) [inline components/Auth/Authform.js]: ????

File: components/Card/Cardgrid.js [hunk 7]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Card from './Card';
+import classes from './Cardgrid.module.css';
+
+function Cardgrid(props) {
+  const {cards} = props;
```
Comment (AntonovaNastya) [inline components/Card/Cardgrid.js:6]: const { cards } = props;

File: components/Card/Cardgrid.js [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+
+  return (
+    <div className={classes.container}>
+      <h2>Товары</h2>
+      <div className={classes.grid}>
+        {cards.map((cards) => (
+          <Card key={cards.arcticle} id={cards.id} cards={cards} />
```
Comment (AntonovaNastya) [inline components/Card/Cardgrid.js]: key={cards.id}

File: components/Carddetail/Carddescription.js [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+            <li>цвет: синий</li>
+            <li>эластичный хлопок</li>
+            <li>эффект потертости</li>
+            <li>узкий крой</li>
+          </ul>
+        </div>
+      </div>
+      <div className={classes.ditails}>
+        <div className={classes.composition}>
+          <h4>Состав</h4>
```
Comment (AntonovaNastya) [inline components/Carddetail/Carddescription.js]: const description = [
{
title: "Состав",
value: "sadsadsad"
},
....]

File: components/Carddetail/Carddescription.module.css [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+.ditails_title {
+  text-align: left;
+}
+
+.ditails {
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 70px;
+  /* justify-content: start;
```
Comment (AntonovaNastya) [inline components/Carddetail/Carddescription.module.css]: !

File: components/Carddetail/Carddetail.js [hunk 11]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Select from '../UI/Select';
+import classes from '../Carddetail/Carddetail.module.css';
+import Blackbtn from '../UI/Blackbtn';
+import Whitebtn from '../UI/Whitebtn';
+import Image from 'next/image';
+import Carddescription from './Carddescription';
+
+function Carddetail(props) {
```
Comment (AntonovaNastya) [inline components/Carddetail/Carddetail.js]: function CardDetail ({ title, description, price ...}) {

File: components/HomePage/Contact/Contactform.js [hunk 12]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```
Comment (AntonovaNastya) [inline components/HomePage/Contact/Contactform.js]: import React, {useState, useRef} from 'react';

File: components/HomePage/Main.js [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+    props: {
+      maindata: maindata.map((maindata) => ({
+        image: maindata.image,
+        title: maindata.title,
+      })),
+    },
+    revalidate: 1,
+  };
+}
+function Main() {
```
Comment (AntonovaNastya) [inline components/HomePage/Main.js]: add space above

File: components/HomePage/Main.js [hunk 14]
```diff
@@ -0,0 +1,71 @@
...
+  return (
+    <>
+      {maindata.map((maindata) => (
+        <div className={`${classes.container} ${maindata.style}`}>
```
Comment (AntonovaNastya) [inline components/HomePage/Main.js]: add key

File: components/HomePage/Сategories/СategoriesGrid.js [hunk 15]
```diff
@@ -0,0 +1,23 @@
...
+  const data = props.actualdata;
+
+  return (
+    <Fragment>
+      <div className={classes.title}>
+        <h2>Категории</h2>
+      </div>
+      <div className={classes.container}>
+        {data.map((data) => (
+          <СategoriesItem actualdata={data} />
```
Comment (AntonovaNastya) [inline components/HomePage/Сategories/СategoriesGrid.js]: add key

File: components/Layout/Footer/Footer.js [hunk 16]
```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import Link from 'next/link';
+
+import classes from './Footer.module.css';
+
+const footerData = [
```
Comment (AntonovaNastya) [inline components/Layout/Footer/Footer.js]: move constants to a separate file

File: components/Layout/Footer/Footer.js [hunk 17]
```diff
@@ -0,0 +1,119 @@
...
+          </a>
+          <a
+            href="tel:+1234567890"
+            className={classes.footer_contact_numbers_item}>
+            + 375-33-351-58-42{' '}
+          </a>
+        </div>
+        <div className={classes.footer_contact_icons}>
+          {iconsData.map((iconsData) => (
+            <a href={iconsData.href}>
```
Comment (AntonovaNastya) [inline components/Layout/Footer/Footer.js]: add key

File: components/Layout/Footer/Footer.js [hunk 18]
```diff
@@ -0,0 +1,119 @@
...
+                src={iconsData.img}
+                className={classes.footer_contact_icon}
+                alt=""
+              />{' '}
+            </a>
+          ))}
+        </div>
+      </div>
+      {footerData.map((footerData) => (
+        <nav className={classes.footer_nav}>
```
Comment (AntonovaNastya) [inline components/Layout/Footer/Footer.js]: add key

File: components/Layout/Header/Header.js [hunk 19]
```diff
@@ -0,0 +1,48 @@
...
+        <div className={classes.header__logo}>
+          <Link href="/">
+            <img src="https://mir-s3-cdn-cf.behance.net/project_modules/disp/17652c15837939.562979668f0e4.jpg"></img>
+          </Link>
+        </div>
+        <div className={classes.header__icon}>
+          <div className={classes.header__icon_items}>
+            <div className={classes.header__icon_item}>
+              {iconData.map((iconData) => (
+                <Link href={iconData.href}>
```
Comment (AntonovaNastya) [inline components/Layout/Header/Header.js]: add key

File: components/Layout/Navigation/Nav/Nav.js [hunk 20]
```diff
@@ -0,0 +1,31 @@
...
+    {href: '/categories', title: 'Категории'},
+    {href: '/wear', title: 'Одежда'},
+    {href: '/delivery', title: 'Доставка'},
+    {href: '/contacts', title: 'Контакнты'},
+  ];
+  return (
+    <header className={classes.header}>
+      <div key={navdata.title} className={classes.nav_title}>
+        {navdata.map((navdata) => (
+          <nav>
```
Comment (AntonovaNastya) [inline components/Layout/Navigation/Nav/Nav.js]: add key

File: components/Layout/Navigation/SideNav/Sidenav.js [hunk 21]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Link from 'next/link';
+import classes from '../SideNav/./Sidenav.module.css';
+
+function Sidenav() {
+  const navdata = [
```
Comment (AntonovaNastya) [inline components/Layout/Navigation/SideNav/Sidenav.js]: move outside component

File: components/UI/Select.js [hunk 22]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import classes from './Select.module.css';
+
+function Select() {
```
Comment (AntonovaNastya) [inline components/UI/Select.js]: function Select({options = [], defaultValue, value })

File: components/UI/Select.js [hunk 23]
```diff
@@ -0,0 +1,21 @@
...
+  return (
+    <div>
+      <div className={classes.custom_select}>
+        <select>
+          <option value="0">Выберите размер:</option>
+          <option value="1">Xs</option>
```
Comment (AntonovaNastya) [inline components/UI/Select.js]: <option value={option.value}>{option.label}</option>

File: components/UI/Whitebtn.js [hunk 24]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import classes from './Whitebtn.module.css';
+
+function Whitebtn({children}) {
+  return (
+    <div>
+      <div className={`${classes.btn} ${classes.btn_dark}`}>{children}</div>
```
Comment (AntonovaNastya) [inline components/UI/Whitebtn.js]: button

File: components/HomePage/Main.js [hunk 25]
```diff
@@ -0,0 +1,71 @@
...
+    image: '/image/main2.webp',
+    title: 'Подарок как искусство',
+    text: 'Для нового осенне-зимнего сезона Burberry впервые сделали из габардина не только тренчи, но и технологичные пуховики и жилеты. В честь выхода коллекции мы объединились с британским брендом и платформой DressX, которая создает виртуальную одежду, чтобы отправить новинки в путешествие. Оцените лучшие модели и узнайте больше о жизни вдали от большого города и Burberry от героев нашей съемки.',
+    style: '{clases.reverse}',
+  },
+];
+
+export async function getStaticProps() {
+  const client = await MongoClient.connect(
+    'mongodb+srv://vit:qwerty123@cluster0.6sdaa.mongodb.net/maininfo?retryWrites=true&w=majority',
```
Comment (AntonovaNastya) [inline components/HomePage/Main.js]: use utils function('maininfo') => (collectionName)
`mongodb+srv://vit:qwerty123@cluster0.6sdaa.mongodb.net/${collectionName}?retryWrites=true&w=majority`

File: lib/db.js [hunk 26]
```diff
@@ -0,0 +1,8 @@
+import {MongoClient} from 'mongodb';
+
+export async function connectToDatabase() {
```
Comment (AntonovaNastya) [inline lib/db.js:3]: use props

File: pages/[goodsId]/index.js [hunk 27]
```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Carddetail from '../../components/Carddetail/Carddetail';
+
+import {MongoClient, ObjectId} from 'mongodb';
+
+export async function getStaticPaths() {
+  const client = await MongoClient.connect(
+    'mongodb+srv://vit:qwerty123@cluster0.6sdaa.mongodb.net/goods?retryWrites=true&w=majority',
```
Comment (AntonovaNastya) [inline pages/[goodsId]/index.js]: use connectToDatabase util

File: pages/[goodsId]/index.js [hunk 28]
```diff
@@ -0,0 +1,67 @@
...
+    paths: goods.map((goods) => ({
+      params: {goodsId: goods._id.toString()},
+    })),
+  };
+}
+
+export async function getStaticProps(context) {
+  const goodsId = context.params.goodsId;
+  const client = await MongoClient.connect(
+    'mongodb+srv://vit:qwerty123@cluster0.6sdaa.mongodb.net/goods?retryWrites=true&w=majority',
```
Comment (AntonovaNastya) [inline pages/[goodsId]/index.js]: use connectToDatabase util

File: pages/api/auth/sighnup.js [hunk 29]
```diff
@@ -0,0 +1,30 @@
+// import {connectToDatabase} from '../../../lib/db';
```
Comment (AntonovaNastya) [inline pages/api/auth/sighnup.js]: !!!!!!

File: pages/api/email.js [hunk 30]
```diff
@@ -0,0 +1,22 @@
+import {MongoClient} from 'mongodb';
+
+async function handler(req, res) {
+  if (req.method === 'POST') {
+    const data = req.body;
+    const {email} = data;
+
+    const client = await MongoClient.connect(
+      'mongodb+srv://vit:qwerty123@cluster0.6sdaa.mongodb.net/email?retryWrites=true&w=majority',
```
Comment (AntonovaNastya) [inline pages/api/email.js]: use connectToDatabase util

File: pages/categories/index.js [hunk 31]
```diff
@@ -0,0 +1,30 @@
...
+
+function index(props) {
+  return <СategoriesGrid actualdata={props.actualdata}></СategoriesGrid>;
+}
+
+export default index;
+
+export async function getStaticProps() {
+  const client = await MongoClient.connect(
+    'mongodb+srv://vit:qwerty123@cluster0.6sdaa.mongodb.net/main?retryWrites=true&w=majority',
```
Comment (AntonovaNastya) [inline pages/categories/index.js:13]: use connectToDatabase util

File: pages/contacts/index.js [hunk 32]
```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import classes from './index.module.css';
+import Link from 'next/dist/client/link';
```
Comment (AntonovaNastya) [inline pages/contacts/index.js:2]: 2 line

File: pages/contacts/index.js [hunk 33]
```diff
@@ -0,0 +1,93 @@
...
+          <div className={classes.footer_logo_social}>
+            <a className={classes.footer_logo_social_item} href="viper.com">
+              Viber
+              <img
+                src="./viber-svgrepo-com.svg"
+                className={classes.footer_logo_social_item}
+                alt=""
+              />{' '}
+            </a>
+            <a className={classes.footer_logo_social_item}>
```
Comment (AntonovaNastya) [inline pages/contacts/index.js:31]: add margin-right and delete {' '}

File: pages/delivery/delivery.module.css [hunk 34]
```diff
@@ -0,0 +1,44 @@
...
+  width: 30%;
+}
+
+.container table {
+  width: 100%;
+  border: 1px solid #dddddd;
+  border-collapse: collapse;
+}
+
+/* .container table tr {
```
Comment (AntonovaNastya) [inline pages/delivery/delivery.module.css]: !

File: pages/delivery/index.js [hunk 35]
```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import Sidenav from '../../components/Layout/Navigation/Sidenav/Sidenav';
+import classes from '../delivery/delivery.module.css';
```
Comment (AntonovaNastya) [inline pages/delivery/index.js:3]: import React from 'react'; // node-moduls

import Sidenav from 'components/Layout/Navigation/Sidenav/Sidenav'; //absolute path

import classes from '../delivery/delivery.module.css'; // in the same folder (delivery)

File: pages/delivery/index.js [hunk 36]
```diff
@@ -0,0 +1,95 @@
...
+          </tr>
+          <tr>
+            <td>Бесплатно</td>
+            <td>Банковской картой онлайн</td>
+          </tr>
+        </table>
+        <h2>Сроки доставки</h2>
+        <table>
+          <tr>
+            <th>Регион</th>
```
Comment (AntonovaNastya) [inline pages/delivery/index.js:45]: use map for data

File: pages/index.js [hunk 37]
```diff
@@ -1,18 +1,70 @@
 import Head from 'next/head';
-import Nav from '../components/Navigation/Nav';
 import styles from '../styles/Home.module.css';
-import Card from '../components/Card/Card';
+import Main from '../components/HomePage/Main';
+import {MongoClient} from 'mongodb';
+import {Fragment} from 'react';
+import СategoriesGrid from '../components/HomePage/Сategories/СategoriesGrid';
+import Cardgrid from '../components/Card/Cardgrid';
+import Contactform from '../components/HomePage/Contact/Contactform';
```
Comment (AntonovaNastya) [inline pages/index.js]: import {Fragment} from 'react';
import {MongoClient} from 'mongodb';
import Head from 'next/head';

import СategoriesGrid from '../components/HomePage/Сategories/СategoriesGrid';
import Cardgrid from '../components/Card/Cardgrid';
import Contactform from '../components/HomePage/Contact/Contactform';
import Nav from '../components/Navigation/Nav';
import Card from '../components/Card/Card';
import Main from '../components/HomePage/Main';

import styles from '../styles/Home.module.css';

File: pages/index.js [hunk 38]
```diff
@@ -1,18 +1,70 @@
...
+        <СategoriesGrid actualdata={props.actualdata}></СategoriesGrid>
+        <Contactform addEmail={addEmailHandler}></Contactform>
+      </Fragment>
     </div>
   );
 }
+
+export async function getStaticProps() {
+  const client = await MongoClient.connect(
+    'mongodb+srv://vit:qwerty123@cluster0.6sdaa.mongodb.net/goods?retryWrites=true&w=majority',
```
Comment (AntonovaNastya) [inline pages/index.js]: use connectionDB

File: pages/wear/index.js [hunk 39]
```diff
@@ -0,0 +1,39 @@
...
+  return (
+    <div>
+      <Cardgrid cards={props.actualdata}></Cardgrid>
+    </div>
+  );
+}
+
+export async function getStaticProps() {
+  const client = await MongoClient.connect(
+    'mongodb+srv://vit:qwerty123@cluster0.6sdaa.mongodb.net/goods?retryWrites=true&w=majority',
```
Comment (AntonovaNastya) [inline pages/wear/index.js]: use connectionDB

File: components/Card/Card.js (hunk 2 above)
Comment (vitalypuzanov) [reply components/Card/Card.js]: +

File: components/Card/Card.js (hunk 3 above)
Comment (vitalypuzanov) [reply components/Card/Card.js]: +

File: components/Card/Card.module.css (hunk 5 above)
Comment (vitalypuzanov) [reply components/Card/Card.module.css]: +

File: components/Card/Cardgrid.js (hunk 8 above)
Comment (vitalypuzanov) [reply components/Card/Cardgrid.js]: +

File: components/Carddetail/Carddescription.module.css (hunk 10 above)
Comment (vitalypuzanov) [reply components/Carddetail/Carddescription.module.css]: +

File: components/HomePage/Contact/Contactform.js (hunk 12 above)
Comment (vitalypuzanov) [reply components/HomePage/Contact/Contactform.js]: +

File: components/HomePage/Main.js (hunk 13 above)
Comment (vitalypuzanov) [reply components/HomePage/Main.js]: +

File: pages/delivery/delivery.module.css (hunk 34 above)
Comment (vitalypuzanov) [reply pages/delivery/delivery.module.css]: +

File: pages/contacts/index.js (hunk 33 above)
Comment (vitalypuzanov) [reply pages/contacts/index.js:31]: +

File: pages/index.js (hunk 37 above)
Comment (vitalypuzanov) [reply pages/index.js]: +

File: components/Card/Card.js [hunk 40]
```diff
@@ -1,29 +1,30 @@
 import React from 'react';
+import Image from 'next/dist/client/image';
+import {useRouter} from 'next/router';
+
 import classes from './Card.module.css';
 
-function Card() {
+function Card(props) {
+  const router = useRouter();
+  const {image, title, description, price} = props.cards;
```
Comment (AntonovaNastya) [inline components/Card/Card.js:9]: function Card({ cards }) {
  const router = useRouter();
  const { image, title, description, price } = cards;

File: components/Card/Card.js [hunk 41]
```diff
@@ -1,29 +1,30 @@
...
+
+  function showDetailsHandler() {
```
Comment (AntonovaNastya) [inline components/Card/Card.js]: const showDetailsHandler = () => {

File: components/Card/Cardgrid.js (hunk 7 above)
Comment (AntonovaNastya) [inline components/Card/Cardgrid.js]:
```
const CardGrid = ({ cards }) => (
    <div className={classes.container}>
      <h2>Товары</h2>
      <div className={classes.grid}>
        {cards.map((cards) => (
          <Card key={cards.id} id={cards.id} cards={cards} />
        ))}
      </div>
    </div>
  );
```

File: components/Carddetail/Carddescription.js [hunk 42]
```diff
@@ -0,0 +1,57 @@
...
+    title: 'Артикул дизайнера',
+    value: 'XMD001',
+  },
+  {
+    title: 'Модель и образ',
+    value: 'Рост модели: 1,86 м.',
+  },
+];
+
+function Carddescription(props) {
```
Comment (AntonovaNastya) [inline components/Carddetail/Carddescription.js]:
```
const CardDescription = ({ image }) => (
<div className={classes.container}>
.....
</div>
);
```

File: components/Carddetail/Carddescription.module.css [hunk 43]
```diff
@@ -0,0 +1,25 @@
+.container {
+  position: relative;
```
Comment (AntonovaNastya) [inline components/Carddetail/Carddescription.module.css]: Are you sure that `position: relative;` and `top: 0;` are necessary?

File: components/HomePage/Contact/Contactform.js [hunk 44]
```diff
@@ -0,0 +1,44 @@
+import React, {useState, useRef} from 'react';
+
+import Blackbtn from '../../UI/Blackbtn';
+import classes from './Contactform.module.css';
+
+function Contactform(props) {
```
Comment (AntonovaNastya) [inline components/HomePage/Contact/Contactform.js]: function ContactForm ({ addEmail }) {

File: components/HomePage/Main.js [hunk 45]
```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Image from 'next/image';
+import classes from './Main.module.css';
+import Blackbtn from '../UI/Blackbtn';
+import {connectToDatabase} from '../../lib/db';
+// import {MongoClient} from 'mongodb';
```
Comment (AntonovaNastya) [inline components/HomePage/Main.js]: !

File: components/HomePage/Main.js [hunk 46]
```diff
@@ -0,0 +1,79 @@
...
+      maindata: maindata.map((maindata) => ({
+        image: maindata.image,
+        title: maindata.title,
+      })),
+    },
+    revalidate: 1,
+  };
+}
+
+function Main() {
```
Comment (AntonovaNastya) [inline components/HomePage/Main.js]: const Main = () => maindata.map((maindata) => (
        <div>
        ....
        </div
      );

File: components/HomePage/Сategories/СategoriesGrid.js [hunk 47]
```diff
@@ -0,0 +1,23 @@
+import React, {Fragment} from 'react';
+import СategoriesItem from './СategoriesItem';
+
+import classes from './СategoriesGrid.module.css';
+
+function СategoriesGrid(props) {
```
Comment (AntonovaNastya) [inline components/HomePage/Сategories/СategoriesGrid.js]:
```
const СategoriesGrid = ({ actualdata })  => (
<Fragment>
      <div className={classes.title}>
        <h2>Категории</h2>
      </div>
      <div className={classes.container}>
        {actualdata.map((data) => (
          <СategoriesItem key={data.image} actualdata={data} />
        ))}
      </div>
    </Fragment>
)
```

File: components/HomePage/Сategories/СategoriesItem.js [hunk 48]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Image from 'next/image';
+import classes from './СategoriesItem.module.css';
+import Blackbtn from '../../UI/Blackbtn';
+
+function СategoriesItem(props) {
```
Comment (AntonovaNastya) [inline components/HomePage/Сategories/СategoriesItem.js]: function СategoriesItem({ actualdata }) {
  const { image, title } = actualdata;

File: components/Layout/Footer/Footer.module.css [hunk 49]
```diff
@@ -0,0 +1,82 @@
...
+.footer_items img {
+  width: 30px;
+}
+
+.footer_nav ul {
+  padding-left: 0;
+}
+
+.footer_nav li {
+  position: relative;
```
Comment (AntonovaNastya) [inline components/Layout/Footer/Footer.module.css:61]: ?

File: components/Layout/Footer/Footerdata.js [hunk 50]
```diff
@@ -0,0 +1,29 @@
+export const footerData = [
+  {
```
Comment (AntonovaNastya) [inline components/Layout/Footer/Footerdata.js:2]:
```
export const footerData = [
  {
    id: '1',
    title: 'ПОКУПАТЕЛЯМ',
    menu: [
      {
        url: '/delivery',
        name: 'Доставка'
      }
    ],
  },
];
```

File: components/Layout/Footer/Footer.js [hunk 51]
```diff
@@ -0,0 +1,83 @@
...
+          ))}
+        </div>
+      </div>
+      {footerData.map((footerData) => (
+        <nav key={footerData.id} className={classes.footer_nav}>
+          <div>
+            <h3>{footerData.title}</h3>
+          </div>
+          <ul>
+            <Link href="/delivery">
```
Comment (AntonovaNastya) [inline components/Layout/Footer/Footer.js]:
```
{footerData.menu.map(menuItem => (
       <Link href={menuItem.url}">
              <li>{menuItem.name}</li>
        </Link>
))}
```

File: components/Layout/Navigation/Nav/Nav.js [hunk 52]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Link from 'next/link';
+import classes from '../Nav/Nav.module.css';
+import Search from '../Search/Search';
+
+function Nav() {
+  const navdata = [
```
Comment (AntonovaNastya) [inline components/Layout/Navigation/Nav/Nav.js]: move outside component

File: components/Layout/Navigation/Nav/Nav.js [hunk 53]
```diff
@@ -0,0 +1,31 @@
...
+    {href: '/categories', title: 'Категории', id: '1'},
+    {href: '/wear', title: 'Одежда', id: '2'},
+    {href: '/delivery', title: 'Доставка', id: '3'},
+    {href: '/contacts', title: 'Контакнты', id: '4'},
+  ];
+  return (
+    <header className={classes.header}>
+      <div key={navdata.title} className={classes.nav_title}>
```
Comment (AntonovaNastya) [inline components/Layout/Navigation/Nav/Nav.js]: key is an unnecessary

File: components/Layout/Navigation/SideNav/Sidenav.js [hunk 54]
```diff
@@ -0,0 +1,28 @@
...
+  {href: '/wear', title: 'ОПЛАТА'},
+  {href: '/delivery', title: 'ВОЗВРАТ'},
+  {href: '/contacts', title: 'ЧАСТЫЕ ВОПРОСЫ'},
+  {href: '/contacts', title: 'КОНТАКТЫ'},
+];
+
+function Sidenav() {
+  return (
+    <div key={navdata.title} className={classes.nav_title}>
+      {navdata.map((navdata) => (
```
Comment (AntonovaNastya) [inline components/Layout/Navigation/SideNav/Sidenav.js]:
```
     <ul>
      {navdata.map((navdata) => (
        <nav key={navdata.title}>
            <li>
              <Link href={navdata.href}>{navdata.title}</Link>
            </li>
        </nav>
      ))}
       </ul>
```

File: components/Layout/Navigation/SideNav/Sidenav.js [hunk 55]
```diff
@@ -0,0 +1,28 @@
...
+  {href: '/sale', title: 'ДОСТАВКА И САМОВЫВОЗ'},
+  {href: '/wear', title: 'ОПЛАТА'},
+  {href: '/delivery', title: 'ВОЗВРАТ'},
+  {href: '/contacts', title: 'ЧАСТЫЕ ВОПРОСЫ'},
+  {href: '/contacts', title: 'КОНТАКТЫ'},
+];
+
+function Sidenav() {
+  return (
+    <div key={navdata.title} className={classes.nav_title}>
```
Comment (AntonovaNastya) [inline components/Layout/Navigation/SideNav/Sidenav.js]: remove key

File: components/UI/Blackbtn.js [hunk 56]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import classes from './Blackbtn.module.css';
+
+function Blackbtn({children}) {
```
Comment (AntonovaNastya) [inline components/UI/Blackbtn.js]: BlackButton

File: components/UI/Whitebtn.js [hunk 57]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import classes from './Whitebtn.module.css';
+
+function Whitebtn({children}) {
```
Comment (AntonovaNastya) [inline components/UI/Whitebtn.js]: WhiteButton

File: pages/delivery/index.js [hunk 58]
```diff
@@ -0,0 +1,84 @@
...
+  {city: 'Москва', data: '1-2 дня', price: '350', id: '3'},
+  {city: 'Москва', data: '1-2 дня', price: '350', id: '4'},
+  {city: 'Москва', data: '1-2 дня', price: '350', id: '5'},
+  {city: 'Москва', data: '1-2 дня', price: '350', id: '6'},
+  {city: 'Москва', data: '1-2 дня', price: '350', id: '7'},
+  {city: 'Москва', data: '1-2 дня', price: '350', id: '8'},
+  {city: 'Москва', data: '1-2 дня', price: '350', id: '9'},
+];
+
+function index() {
```
Comment (AntonovaNastya) [inline pages/delivery/index.js]: ??????

File: pages/[goodsId]/index.js (hunk 27 above)
Comment (AntonovaNastya) [reply pages/[goodsId]/index.js]: const client = connectToDatabase('goods');

File: components/Layout/Footer/Footer.module.css (hunk 49 above)
Comment (vitalypuzanov) [reply components/Layout/Footer/Footer.module.css:61]: это для анимации

File: lib/db.js (hunk 26 above)
Comment (vitalypuzanov) [reply lib/db.js:3]: i wiil add all data in one database and now not nessesary to use props

**Review by AntonovaNastya — APPROVED** [review]

**Pull request merged**